iqpuzzle.txt

```diff
@@ -1,15 +1,15 @@
 [Section]
 Name = iQPuzzle
-Version = 1.2.0
+Version = 1.2.1
 License = Open Source (GPLv3)
 Description = A diverting I.Q. challenging pentomino puzzle
 Size = 23.0 MiB
```

Comment:
`Size` is obsolete if `SizeBytes` is used :)

Reply:
Many thanks for the information! I'll correct it.
The [RAPPS article](https://reactos.org/wiki/RAPPS) in the wiki confused me a little bit. I thought 'Size' = of the program (after extraction/installation) and 'SizeBytes' = Size of the download (packed installer).